bin/armbian-interface

```diff
@@ -36,7 +36,7 @@
 #
 
 ## DIRECTORY variable to the absolute path of the script's directory
-directory="$(dirname "$(readlink -f "$0")")"
+# directory="$(dirname "$(readlink -f "$0")")"
```

Comment:
This is not needed anymore?

Reply:
Sellcheck says it is not needed, Since armbian-interface is an indepented app I would agree. 

at one time it was `filename=$directory...` and was forgotten after the chenge. 

Succefuly tested with the `echo "hi" | arbian-interfacce -o`